test/fixtures/base/test-img/img.twig

```diff
@@ -0,0 +1,3 @@
+<!-- PATTERN START - /pattern-library/patterns/base/img -->
```

Comment:
this file is related to img component. I assume, It's for checking img markup, but wouldn't test-purpose kind of files leave in the same repo which they are related to?